### PR TITLE
Cover/cmds/core/gpgv

### PR DESCRIPTION
**Pull request by ChriMarMe**

### CODE REVIEW

File: cmds/core/gpgv/gpgv.go [hunk 1]
```diff
@@ -120,3 +92,40 @@ func verifyDetachedSignature(key *packet.PublicKey, contentf, sigf io.Reader) er
 	}
 	return err
 }
+
+func runGPGV(w io.Writer, verbose bool, args []string) error {
```
Comment (llogen) [inline cmds/core/gpgv/gpgv.go:96]: I don't think you have to pass verbose. Isn't it a global variable you can also set in your test file? Would be one argument less.

File: cmds/core/gpgv/gpgv_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		{
+			name:    "InvalidSignatureFile",
+			args:    []string{"testdata/key.pub", "testdata/datafile.txt", "testdata/datafile.txt"},
+			verbose: true,
+			wantErr: fmt.Errorf("tag byte does not have MSB set"),
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			var buf bytes.Buffer
+
```
Comment (llogen) [inline cmds/core/gpgv/gpgv_test.go:47]: verbose = tt.verbose

**Review by llogen — CHANGES_REQUESTED** [review]
Just one minor thing. LGTM :)

File: cmds/core/gpgv/gpgv_test.go (hunk 2 above)
Comment (ChriMarMe) [reply cmds/core/gpgv/gpgv_test.go:47]: jup :D i forgot

File: cmds/core/gpgv/gpgv.go [hunk 3]
```diff
@@ -120,3 +92,40 @@ func verifyDetachedSignature(key *packet.PublicKey, contentf, sigf io.Reader) er
...
+
+	key, err := readPublicSigningKey(keyf)
+	if err != nil {
+		return fmt.Errorf("key: %v ", err)
+	}
+
+	if err = verifyDetachedSignature(key, contentf, sigf); err != nil {
+		return fmt.Errorf("verify: %v", err)
+	}
+	fmt.Fprintf(w, "%s", "OK")
```
Comment (MDr164) [inline cmds/core/gpgv/gpgv.go:122]: nit: Doesn't `fmt.Fprint(w, "OK")` also work here?

**Pull request closed without merging**